### PR TITLE
udpate packages & improvements

### PR DESCRIPTION
**Pull request by paranoida**
- update packages
- small improvements
- cleanup

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -2,9 +2,9 @@
   "name": "fe_task",
   "private": true,
   "dependencies": {
-    "jquery": "~2.1.1",
-    "jasmine": "~2.0.4",
-    "jasmine-jquery": "~2.0.5"
+    "jquery": "~2.1.4",
```
Comment (codejet) [inline bower.json]: shouldn't that be gone altogether?

Comment (paranoida) [inline bower.json]: true.

**Comment by codejet** [comment]
:+1:

**Comment by iGEL** [comment]
Is this repo public on purpose?

**Comment by codejet** [comment]
@iGEL is watching you...

**Comment by codejet** [comment]
btw: yes it is :smirk:

**Comment by paranoida** [comment]
@iGEL what are you doing here!!! It's public but there is nothing inside - just environment

**Comment by iGEL** [comment]
Ok, cool


(I got notifications about your PR, will unsubscribe now :wink:)

**Comment by paranoida** [comment]
@codejet I broke livereloading.. fixing it now

**Comment by duksis** [comment]
oh man! this Boner thing

**Comment by paranoida** [comment]
Fixed. @duksis @iGEL please review

**Comment by duksis** [comment]
I see you have descriptive commit messages - looks acceptable :+1:

**Comment by lukaszklis** [comment]
Works good, could you add the new favicon? ;)

**Pull request merged**